_includes/includes/ui/keyboard-details.php

```diff
@@ -274,7 +274,7 @@ protected static function WriteiPhoneBoxes() {
     protected static function WriteiPadBoxes() {
       global $appstore;
       if (isset(self::$downloads->js)) {
```

Comment:
Why does WriteiPadBoxes have `self::$downloads->js` when the other mobile platforms don't?

Reply:
Because reasons. Will fix.